composer/core/state.py

```diff
@@ -81,7 +81,7 @@ class State(Serializable):
         precision_context ((precision: Precision) -> ContextManager): Function to produce a context manager to mandate precision.
 
         optimizers (types.Optimizers, optional): The optimizers being used to train the model. Multiple optimizers are not currently supported.
-        schedulers (types.Schedulers, optional): The learning rate schedulers, typically wrapped in :class:`ComposableScheduler`.
+        schedulers (types.PytorchScheduler, optional): The learning rate schedulers.
```

Comment:
```suggestion
        schedulers (types.PytorchScheduler, optional): The learning rate scheduler[s].
```
Less implication that there has to be more than one. Also I'm a little confused here because we only allow one scheduler according to the type signature, but the name `schedulers` is plural.

Reply:
See #640.